src/wasm-common.h

```diff
@@ -142,7 +142,7 @@ enum { WASM_USE_NATURAL_ALIGNMENT = 0xFFFFFFFF };
   V(I32, ___, 0, 0x57, I32_CLZ)             \
   V(I32, ___, 0, 0x58, I32_CTZ)             \
   V(I32, ___, 0, 0x59, I32_POPCNT)          \
-  V(I32, ___, 0, 0x5a, I32_EQZ)             \
+  V(I32, ___, 0, 0x5a, I32_EQZ_I32)         \
```

Comment:
I'd rather keep the names consistent with the spec names, so `I32_EQZ` and `I64_EQZ`